### DISCUSSION TITLE
Multi AI Agent Systems with crewAI - Course

### DISCUSSION THREAD

**Discussion by theCyberTech** [crewAIInc/crewAI#1151] [Announcements]
If you haven't already be sure to checkout our course on Deeplearning.AI

[Multi AI Agent Systems with crewAI](https://www.deeplearning.ai/short-courses/multi-ai-agent-systems-with-crewai/)

**Comment by nickzren** [comment]
Very helpful one hour course

**Reply by niradar** [reply]
That "one hour" took me 4 days. The video itself, without practising, is more than 2 hours, so I don't know how you can do it in one hour. Double speed maybe? :)

**Comment by Goldokpa** [comment]
It was a pleasure presenting on How  Ai Agents are reshaping enterprise systems at the just concluded UK CrewAI week, it was filled lots of insights from industry leaders. i cant wait for the next years event
